### PR TITLE
Use restarts fom vk83

### PR DESCRIPTION
**Pull request by MartinDix**
Remove the scripts that process restarts to fix years and instead use processed versions from vk83, #20

No change to results.

### CODE REVIEW

File: scripts/set_restart_year.sh [hunk 1]
```diff
@@ -1,110 +0,0 @@
...
-
-# Load some helper scripts
-SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
-source $SCRIPTDIR/utils.sh
-
-# Starting year
-start_year=$1
-
-# Set the restart year in the atmosphere and ice namelists
-set_um_start_year $start_year
```
Comment (blimlim) [inline scripts/set_restart_year.sh:37]: Noticed that these sectionsthe control directory namelists. It looks like they are already set to the right values of 1850-01-01 though so everything looks ok.

**Review by blimlim — APPROVED** [review]
Everything looks good to me! Just the small issue about the read permissions on the atmosphere restart file, though that doesn't require any changes to the config.

I just wanted to check whether we'll hang on to these scripts anywhere, and whether we'll need them down the road for setting up new configurations?

File: config.yaml [hunk 2]
```diff
@@ -41,7 +41,7 @@ collate:
    restart: true
    mem: 4GB
 
-restart: /g/data/access/payu/access-esm/restart/pre-industrial
+restart: /g/data/vk83/experiments/inputs/access-esm1p5/historical/restart
```
[the remark this answers is not in the document]
Comment (MartinDix) [reply config.yaml:44]: File permission fixed now.

**Comment by MartinDix** [comment]
The intention is to put these scripts into a stand-alone package that anyone can use to create an historical restart from anywhere in the PI control, #7.

**Pull request merged**